arm-logic/2016-06-01/examples/SessionsCreateOrUpdate.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "Create or update integration account session example",
+  "parameters": {
+    "api-version": "2016-06-01",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "integrationAccountName": "testia123",
+    "sessionName": "testsession123-ICN",
+    "session": {
+      "properties": {
+        "content": "{\"controlNumber\":\"1234\",\"controlNumberChangedTime\":\"2017-02-21T22:30:11.9923759Z\"}"
```

Comment:
I have tested PUT without the location and the request is accepted by the RP. So this is correct.